db/seeds.d/76-job_templates.rb

```diff
@@ -15,8 +15,10 @@
         ForemanSalt.register_rex_feature
       end
       JobTemplate.without_auditing do
-        Dir[File.join("#{ForemanSalt::Engine.root}/app/views/foreman_salt/"\
-                    'job_templates/**/*.erb')].each do |template|
+        dirs = Dir[File.join("#{ForemanSalt::Engine.root}/app/views/foreman_salt/"\
+                    'job_templates/**/*.erb')]
+        first = dirs.select { |d| d =~ /function/ }
```

Comment:
I would prefer to check for "salt_run_function" instead of regex function. Like
d.start_with('salt_run_function')